src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -5,6 +5,7 @@ Release History
 
 2.0.36
```

Comment:
This should bump the minor version, not the patch version, yes?

Reply:
You know we had breaking changes before but we never touched the minor version. I am fine with either way but want to confirm we will pick up this policy moving forward.

Reply:
Let's bump the minor version this time. We should keep doing this from now on.

Reply:
I would bump the minor version, but that's not even proper semantic versioning (though it feels *more right*).  @johanste  do you have a preference?

Reply:
Agree with @troydai

Reply:
So, i have updated to `2.1.0`